common/buf/readv_reader.go

```diff
@@ -134,17 +133,13 @@ func (r *ReadVReader) ReadMultiBuffer() (MultiBuffer, error) {
 	return mb, nil
 }
 
-var useReadv = true
+var useReadv bool
```

Comment:
simply `var useReadv = false` could fix the Switch behavior in init().

seems Readv is enabled if there is a applicable environment, you should require @RPRX for a Disable statement.

Reply:
According the release notes of v1.0.0, I think ReadV should be enabled in all environment.